### PR TITLE
Changes from classifiers to where_clause

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

Classifiers have been changed to a where clause. This PR changes the batch reminders script for the new changes.

# What has changed

Json classifiers changes to sql where clauses. Script updated to handle this.
# How to test?

Run the batch reminder script by following the readme instructions along with the other branches on this ticket built. 
# Links

https://trello.com/c/TnldEmYZ/1023-address-frame-delta-generate-additional-initial-contact-files-13

### CODE REVIEW

**Comment by hbrace** [comment]
[Address Frame Delta - generate additional initial contact files (13)](https://trello.com/c/TnldEmYZ/1023-address-frame-delta-generate-additional-initial-contact-files-13)

**Review by NickGrantX — CHANGES_REQUESTED** [review]
`user_defined_where_clause` is still `classifiers` (or should be) - see other comments on PRs

File: reminder_batch_scheduler/reminder_batch.py [hunk 1]
```diff
@@ -12,7 +11,7 @@
 
 def main(wave: int, starting_batch: int, max_cases: int, action_plan_id: uuid.UUID, insert_rules: bool = False,
          trigger_date_time: datetime = None):
-    wave_classifiers = constants.WAVE_CLASSIFIERS[wave]
+    wave_classifiers = {'where_clause': constants.WAVE_CLASSIFIERS[wave]}
```
Comment (AdamHawtin) [inline reminder_batch_scheduler/reminder_batch.py]: Not sure why this is wrapped in a dictionary, it gets read out of it every place it's used that I could see

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: test/reminder_batch_scheduler/test_reminder_batch.py [hunk 2]
```diff
@@ -158,58 +157,20 @@ def test_build_batch_count_query(wave_classifiers, expected_query, expected_para
...
-        }),
-        'P_QU_H2': json.dumps({
-            'treatment_code': [
-                'HH_LP1W'
-            ],
-            'print_batch': list(range(1, 99)),
-        })
+    (3, [str(i) for i in range(1, 99)], {
+        'P_QU_H1': f'''case_type != 'HI' AND treatment_code IN ('HH_LP1E') AND print_batch IN ('{TEST_BATCH_COUNT}')''',
+        'P_QU_H2': f'''case_type != 'HI' AND treatment_code IN ('HH_LP1W') AND print_batch IN ('{TEST_BATCH_COUNT}')'''
```
Comment (AdamHawtin) [inline test/reminder_batch_scheduler/test_reminder_batch.py]: Not sure why these strings are triple quoted (style only) 💅

**Review by AdamHawtin — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]

**Pull request merged**